### PR TITLE
feat(translations): add french

### PR DESCRIPTION
**Pull request by Olyno**
## What does this PR do?

This pull request introduces the french translation.

## Test Plan

/

## Related PRs and Issues

#1687 

### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?

Yes i did

### CODE REVIEW

**Review by Stig124 — APPROVED** [review]
`Salut` is not that formal, maybe `Bonjour/Bonsoir` would be more appropriate, approving the rest !

**Review by Stig124 — APPROVED** [review]

File: app/config/locale/terms.json [hunk 1]
```diff
@@ -406,7 +406,7 @@
     },
     {
         "term": "countries.fr",
-        "comment": "These country names are used in Appwrite's Locale API. Learn more at https://appwrite.io/docs/client/locale#localeGetCountries",
+        "comment": "Ces noms de pays sont utilisés dans l'API Locale d'Appwrite. Pour en savoir plus, consultez le site https://appwrite.io/docs/client/locale#localeGetCountries.",
```
Comment (christyjacob4) [inline app/config/locale/terms.json:409]: You don't need to modify this file as part of the PR. :slightly_smiling_face:  You can revert this change

File: app/config/locale/translations/fr.json [hunk 2]
```diff
@@ -3,30 +3,30 @@
...
-    "emails.invitation.thanks": "",
-    "emails.invitation.signature": "",
+    "emails.verification.subject": "Vérification du compte",
+    "emails.verification.hello": "Bonjour {{name}}",
+    "emails.verification.body": "Suivez ce lien pour vérifier votre adresse mail.",
+    "emails.verification.footer": "Si vous n'avez pas demandé à vérifier cette adresse mail, vous pouvez ignorer ce message.",
+    "emails.verification.thanks": "Merci",
+    "emails.verification.signature": "Équipe {{project}}",
+    "emails.magicSession.subject": "Connexion",
+    "emails.magicSession.hello": "Bonjour",
```
Comment (christyjacob4) [inline app/config/locale/translations/fr.json:13]: Please use the same punctuation as seen in the English translation https://github.com/appwrite/appwrite/blob/master/app/config/locale/translations/en.json#L13

**Review by christyjacob4 — CHANGES_REQUESTED** [review]

**Review by christyjacob4 — APPROVED** [review]

**Comment by christyjacob4** [comment]
THANK YOU so much! All changes merged 🥳
Please reach out to me on our [Discord server](https://appwrite.io/discord) if you would like to claim your Appwrite swag!

**Pull request merged**